src/python/tests/spt3g.py

```diff
@@ -0,0 +1,551 @@
+# Copyright (c) 2015-2017 by the parties listed in the AUTHORS file.
```

Comment:
Should be 2018-> ?

Reply:
Ok, I should probably recursively regex-check all the files.  Maybe pre-emptively update them to 2019 :-)